### PR TITLE
Fix windows runtime dir

### PR DESCRIPTION
**Pull request by ashley-cui**
Needed to un-break podman win main.

Needs to be vendored into common, then into podman.

### CODE REVIEW

**Comment by ashley-cui** [comment]
Thanks! @giuseppe 

cc @mheon @baude @TomSweeneyRedHat

**Comment by mheon** [comment]
LGTM
@nalind PTAL

File: pkg/homedir/homedir_windows.go [hunk 1]
```diff
@@ -25,8 +26,36 @@ func Get() string {
...
+// If XDG_RUNTIME_DIR is not set, GetRuntimeDir will try to find a suitable
+// directory for the current user.
+//
+// See also https://standards.freedesktop.org/basedir-spec/latest/ar01s03.html
+func GetRuntimeDir() (string, error) {
+	data, err := GetDataHome()
+	if err != nil {
+		return "", err
+	}
+	runtimeDir := filepath.Join(data, "containers", "podman")
```
Comment (nalind) [inline pkg/homedir/homedir_windows.go:59]: Looks like a weird name choice when the code making it lives in this repository.

Comment (giuseppe) [reply pkg/homedir/homedir_windows.go:59]: opened a PR to fix it: https://github.com/containers/storage/pull/1812

Comment (baude) [reply pkg/homedir/homedir_windows.go:59]: i agree ... seems misplaced

**Pull request merged**